### ISSUE TITLE
Sniper Elite 5 (1029690)

### ISSUE THREAD

**Issue by ThisNekoGuy** [ValveSoftware/Proton#6140]
# Compatibility Report
- Name of the game with compatibility issues: Sniper Elite 5
- Steam AppID of the game: 1029690

## System Information
- GPU: RTX 2080 Ti
- Driver/LLVM version: Nvidia ([Open-GPU Kernel Modules](https://github.com/NVIDIA/open-gpu-kernel-modules)) 515.65.01
- Kernel version: [Linux 5.17.2-256-tkg-pds](https://github.com/frogging-family/linux-tkg)
- Filesystem: BTRFS
- Display Server: X11 / Xorg
- Link to full system information report: [gist](https://gist.github.com/ThisNekoGuy/8f65157ef5ee115ed11bae5a9bbcc51e)
- Proton version: Proton 7.0-4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-1029690.log](https://github.com/ValveSoftware/Proton/files/9493578/steam-1029690.log)

## Symptoms 

- The game's launcher renders incorrectly
- The launcher's settings menu doesn't scale to my HiDPI display (3840 x 2160)
- The game launches with EAC support but the cross-platform invite codes don't work, in my experience. (I have an Open NAT type) and I usually don't have a problem with connecting to my friend for games at all (except this one)

## Reproduction

1. Install the EAC runtime from the tools library
2. Install the game
3. Set `Proton 7.0-4` as the used Proton version, if not already
4. Run the game and reach the main menu
5. Start a campaign or PvP multiplayer session
6. Use the game's invite code system to either share it with a player on Xbox (this was what I did) or have the other layer share it with you
7. Join or be joined by the other player
8. Receive an error code

**Comment by joe1inux** [comment]
[steam-1029690.log](https://github.com/ValveSoftware/Proton/files/9847177/steam-1029690.log)
Almost everything is identical, except I don't even make it to the game.  The launcher shows up, (incorrectly formatted), the screen goes black, and then it just crashes out.

**Comment by unclamped** [comment]
I am also experiencing some errors regarding EAC, neither Proton experimental (both stable and bleeding edge) or Proton-GE work, and give the same error.
![image](https://user-images.githubusercontent.com/104658278/198905019-995e48ce-bd33-477c-80b3-8e01cfeeef43.png)
[steam-1029690.log](https://github.com/ValveSoftware/Proton/files/9896991/steam-1029690.log)

**Comment by ThisNekoGuy** [comment]
@unclamped Just to be sure: you do have the EAC runtime installed from the "Tools" category in your Library, right?

**Comment by unclamped** [comment]
> @unclamped Just to be sure: you do have the EAC runtime installed from the "Tools" category in your Library, right?

Yes, tried initializing the game both with it installed and without it. Apparently Sniper Elite uses EOS EAC from what someone told me in the ProtonDB Discord server, although I'm not really sure

**Comment by NTMan** [comment]
Last update breaks the launching the game.

![Screenshot from 2022-11-29 21-43-17](https://user-images.githubusercontent.com/200750/204590448-1cfd0412-de95-4fc5-8ba1-8216aa2654c1.png)

[steam-1029690.log](https://github.com/ValveSoftware/Proton/files/10115183/steam-1029690.log)

**Comment by kisak-valve** [comment]
Hello @NTMan, `wine: Unimplemented function iphlpapi.dll.NotifyNetworkConnectivityHintChange called at address 0000000170032138 (thread 0344), starting debugger...` looks like the line of interest from your log.

**Comment by howels** [comment]
Since the latest giant 40GB patch this has started crashing. Log attached.

[steam-1029690.log](https://github.com/ValveSoftware/Proton/files/10115650/steam-1029690.log)

**Comment by howels** [comment]
> Hello @NTMan, `wine: Unimplemented function iphlpapi.dll.NotifyNetworkConnectivityHintChange called at address 0000000170032138 (thread 0344), starting debugger...` looks like the line of interest from your log.

Same error in my log. Any workaround?

**Comment by jorgicio** [comment]
I thought it was just me, but not, the game borked after the latest update. First it was the EAC, now the entirely game, even with native Vulkan. No workaround found by now. Tried with Proton Experimental and Proton-GE 7-42.

**Comment by howels** [comment]
Really annoying cos I was 50% through the game when that update landed.

**Comment by klti** [comment]
It's working again with Vulkan with the Hotfix Update from 2022-12-05.

**Comment by VladimirMrzv** [comment]
Can confirm, after latest update, now working with proton 7.0.5 and experimental

**Comment by jorgicio** [comment]
Yes, after the latest update the game is working again with Vulkan. However, the EAC issue («hash catalogue not found») is still present.

**Comment by NTMan** [comment]
> However, the EAC issue («hash catalogue not found») is still present.

![Screenshot from 2023-01-07 14-15-26](https://user-images.githubusercontent.com/200750/211143370-15f5f68e-ebf8-4886-b715-1854d89c6bdd.png)

I confirm that anti-cheat didn't work.
[steam-1029690.log](https://github.com/ValveSoftware/Proton/files/10365885/steam-1029690.log)

Although Proton EasyAntiCheat Runtime is installed.
![image](https://user-images.githubusercontent.com/200750/211143480-f921aeef-87c7-438c-91c0-8f528fad8647.png)

**Comment by msmafra** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6140#issuecomment-1330945899

I just installed it and it's not working. Just the launcher works, not the game and Vulkan is greyed out. :(
It grew almost 20Gigabytes in size since the last time I installed it. WOW!

**Comment by jorgicio** [comment]
I don't know if it's due to recent versions of Proton or some update, but multiplayer is now working. I just played a deathmatch without issues. Using Valve Proton 8.0.

**Comment by rhoot** [comment]
My game refused to even pop up the launcher, so I set `PROTON_LOG=1` to get a log file. Turns out Steam had created a 32-bit wine prefix instead of a 64-bit one. After deleting the game's `compatdata` folder and launching the game again it worked.

[steam-1029690.log](https://github.com/ValveSoftware/Proton/files/11711779/steam-1029690.log)

**Comment by kattjevfel** [comment]
Today's update seems to have broken the game on proton, it crashes after pressing play and showing the splash screen for a bit.

[steam-1029690.log](https://github.com/ValveSoftware/Proton/files/11970544/steam-1029690.log)

Seems to be related to EAC, disabling it in the launcher makes the game run with Proton 8.0-2, but even better is that it seems to work just fine even with experimental, though I'll have to wait for later to actually try co-op.

**Comment by jorgicio** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6140#issuecomment-1624013733

It looks that you ran it using DX12, which is kinda broken when using Proton. Try Vulkan version instead.

**Comment by KDERazorback** [comment]
Is seems that with the latest update (The Kraken update) they updated EAC and its now crashing on the splash screen. Uninstalling it and disabling EAC in the launcher options makes the game works again, so its an incompatibility with newer versions of EAC.

**Comment by kattjevfel** [comment]
@jorgicio I was always launching it with vulkan, idk why the logs would show otherwise.

@KDERazorback Launch the game with Proton Experimental, I was able to launch and play the game in co-op for a few hours last night.

**Comment by jorgicio** [comment]
> @jorgicio I was always launching it with vulkan, idk why the logs would show otherwise.
> 
> @KDERazorback Launch the game with Proton Experimental, I was able to launch and play the game in co-op for a few hours last night.

Ok, I confirm that tried launching with Proton Experimental, it works but multiplayer got broken again after can't find the binary .exe. A similar issue happened in Epic version of Dead by Daylight.

**Comment by KDERazorback** [comment]
> @jorgicio I was always launching it with vulkan, idk why the logs would show otherwise.
> 
> @KDERazorback Launch the game with Proton Experimental, I was able to launch and play the game in co-op for a few hours last night.

Doesnt work for me. Tried with a clean compatdata in every scenario possible, tried Proton Experimental and Proton 8.0-2. Also tried GE-Proton8-6 without success. In all cases the game works perfectly on single-player if EAC is disabled on the game launcher.

When EAC is enabled for coop or online multiplayer, the game crashes on the splash screen while "Initializing" after a couple of seconds.

Log doesnt show anything relevant. Renderer was always set to Vulkan on launcher.

Tried to disable mangohud and steam overlay. Still not working.

Was working perfectly before the Kraken update last week with Proton Experimental and EAC enabled for Coop campaign.

I think they updated the EAC runtime and that broke compatibility with Proton.

**Comment by Shmob1** [comment]
Just want to add my 2 cents, for this new EAC issue post Kraken update.
Was playing coop & multiplayer before the Kraken update just fine using Vulkan mode with high & stable fps, with proton exp & 535 Nvidia drivers.

After the update & when EAC is enabled, I get below 30 fps all the time, with large variations in frame time.
If I go offline during a level (which presumably disables EAC), I instantly go back up to my old performance (200+ fps), even during the same level.

**Comment by jorgicio** [comment]
> Just want to add my 2 cents, for this new EAC issue post Kraken update. Was playing coop & multiplayer before the Kraken update just fine using Vulkan mode with high & stable fps, with proton exp & 535 Nvidia drivers.
> 
> After the update & when EAC is enabled, I get below 30 fps all the time, with large variations in frame time. If I go offline during a level (which presumably disables EAC), I instantly go back up to my old performance (200+ fps), even during the same level.

So could you play in MP and co-op after the Kraken update?

**Comment by Shmob1** [comment]
> 

Yes, I am able to play MP & co-op - however, my framerate drops from ~200 offline, to ~30, and the frame times are even worse.

**Comment by jorgicio** [comment]
> > 
> 
> Yes, I am able to play MP & co-op - however, my framerate drops from ~200 offline, to ~30, and the frame times are even worse.

How did you manage it to play MP? Because every time I want to try, it says that anticheat blocks me due to binary not found in the catalogue. I'm using the Vulkan version with Proton Experimental.

Thanks.

**Comment by KDERazorback** [comment]
> > 
> 
> Yes, I am able to play MP & co-op - however, my framerate drops from ~200 offline, to ~30, and the frame times are even worse.

Based on my findings and the (outdated) results in protondb, I'm a little skeptic on this. Maybe you got something installed that us don't? Which OS are you using? GPU and driver version, etc?

I Just tested in a completely fresh install of ZorinOS 16.2 (based on ubuntu LTS) with latest updates, nvidia GPU with latest driver from the 535 version, latest steam, Proton Experimental, no tweaks, no modified drivers, no other programs (neither discord, OBS or anything) and it doesnt work at all. Crashes while initializing EAC if enabled.

If I disable EAC it works perfectly.

Maybe something to do with nvidia itself?

On my installation I have driver 525 and still nothing. My specs:
OS: Zorin OS 16.2
KERNEL: 5.15.0-73-generic
CPU: AMD Ryzen 5 2600X Six-Core
GPU: NVIDIA GeForce RTX 2070
GPU DRIVER: NVIDIA 525.105.17
RAM: 80 GB

**Comment by kattjevfel** [comment]
Been working great with Proton Experimental with EAC enabled, and now with Proton 8.0-3 released it works with that too.

OS: Arch Linux
Kernel: 6.1.39-1-lts
CPU: AMD Ryzen 7 3700X
GPU: AMD Radeon RX 6600 XT (mesa 23.1.4)
RAM: 48GB

**Comment by KDERazorback** [comment]
Finally found a partial solution to the issue.

When launching the game, wine64 was throwing a segmentation fault error on dmesg that immediately crashed the game on the splash screen if EAC is enabled.

Upon investigating, it seems newer versions of EAC also require newer versions of some libs not available on many older distros (like ZorinOS 16.2).

Switched to the latest version of PopOS! with a lot of updated libraries and newer kernel but still the old fashioned 525 nvidia driver and finally the linux portion of EAC loaded properly and was able to actually launch the game with it enabled.

But now the game fails to enter any coop or multiplayer mode from the main menu due to "the game executable" not being found on the [EAC] catalogue. Tried deleting the compatdata again, verifying game files, installing it to the root drive, installing EasyAntiCheat runtime from within Steam but still not working.


It seems it wasnt a compatibility issue with Proton, but EAC requiring very up-to-date libs instead. This explains why @kattjevfel can play in Multiplayer/Coop.

The result is the same tho, not being able to play online since the Kraken update due to an updated EAC.

![Screenshot from 2023-07-23 02-00-40](https://github.com/ValveSoftware/Proton/assets/13314069/7f2d79c6-1ea8-4fee-850e-0c671a454b03)

**Comment by Shmob1** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6140#issuecomment-1646090792
> Replying to https://github.com/ValveSoftware/Proton/issues/6140#issuecomment-1646279707

Sorry for delay in reply, nothing special really. I have a 2080 Super & 4090, both running ubntu 23.04 & both with 535.86.05 from the [graphics driver repo](https://launchpad.net/~graphics-drivers/+archive/ubuntu/ppa).
Proton 8.03 also slightly helped the framerate when EAC is enabled. Other than that I really made no other changes, having EAC proton helper downloaded or not made 0 difference.

**Comment by jorgicio** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6140#issuecomment-1646713086

what kind of libs? I'm EndeavourOS, with recent libraries updated, and tried with Proton Experimental and still can't play MP due to binary cannot be found (the .exe with the Vulkan one).

**Comment by Icexdd** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/6140#issuecomment-1646713086

Same issue on steam deck :(

**Comment by jorgicio** [comment]
Still not working under Endeavour OS, having installed on an external drive. Any fix/workaround? Using Proton 8.0. (I don't want to try several Proton versions in a time, since the game has Denuvo)

**Comment by kisak-valve** [comment]
Sniper Elite 5 (1029690)

Issue transferred from https://github.com/ValveSoftware/Proton/issues/8482.
@ZBKX posted on 2025-02-26T18:52:38:

# Compatibility Report
- Name of the game with compatibility issues:Sniper Elite 5 (1029690)
- Steam AppID of the game:1029690

## System Information
- GPU: AMD Radeon 7600M XT
- Video driver version: amdgpu
- Kernel version:6.13.4-1-default 
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: experimental-9.0-20250218 / hotfix-20250221-2 / proton-9.0-4d


## I confirm:
- [ ] that I haven't found an existing compatibility report for this game.
- [ *] that I have checked whether there are updates for my system available.


Proton logs:
[steam-1029690 experimental-9.0-20250218.log](https://github.com/user-attachments/files/18991064/steam-1029690.experimental-9.0-20250218.log)

[steam-1029690 hotfix-20250221-2.log](https://github.com/user-attachments/files/18991107/steam-1029690.hotfix-20250221-2.log)

[steam-1029690 proton-9.0-4d.log](https://github.com/user-attachments/files/18991114/steam-1029690.proton-9.0-4d.log)

## Symptoms 
1.When hitting "Play" on a game the window "Preparing to Launch" appears, for some seconds, then disappears and the button turns to the "play" button again, without launching the game.

## Reproduction
1.Download Sniper Elite 5 
2.Proton choose 9.0
3.Hitting "Play"
4.Window disappears and the button turns to the "play" button again
5. Proton choose Experimental
6.Hitting "Play"
7.Window disappears and the button turns to the "play" button again
8. Proton choose Hotfix
9.Hitting "Play"
10.Window disappears and the button turns to the "play" button again

## Tried solutions
1.Remount noexec file system path
2.Stop AppArmor
3.Clear Wine cache
4.Reinstall the game and proton

**Comment by kisak-valve** [comment]
Hello @ZBXX since you mentioned noexec, that implies you might be using a secondary Steam library folder location with restrictions. If that's true, what filesystem are you using there and is Proton itself on that partition? Does any game work with those Proton versions?

**Comment by ZBKX** [comment]
I used the system default installation location.My filesystem is Btrfs.Proton is on my main partition.Other games also can't run properly on Pronton.

**Comment by ZBKX** [comment]
> Hello @ZBXX since you mentioned noexec, that implies you might be using a secondary Steam library folder location with restrictions. If that's true, what filesystem are you using there and is Proton itself on that partition? Does any game work with those Proton versions?

I used the system default installation location.My filesystem is Btrfs.Proton is on my main partition.Other games also can't run properly on Pronton.

**Comment by kisak-valve** [comment]
Half black screen reappearing on Sniper Elite 5

Issue transferred from https://github.com/ValveSoftware/SteamOS/issues/2483.
@hiltonsimon posted on 2026-06-19T04:40:07:

#### Your system information

* Steam client version: SteamOS Holo
* SteamOS version: 3.8.10
* Opted into Steam client beta?: No 
* Opted into SteamOS beta?: No
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
Open Sniper Elite 5 and half the screen is blacked out like described here https://www.reddit.com/r/SteamDeck/comments/zyxdnd/sniper_elite_5_screen_half_black_after_os_344/

and here https://steamcommunity.com/app/1029690/discussions/0/693122745159955546/?l=dutch

Should be able to open the game and have it all there. 

#### Steps for reproducing this issue:

1. Open Sniper Elite 5
2. Screen is blacked out.

**Comment by ZereoX** [comment]
@hiltonsimon @phisiks The black screen and visual corruption you reported on SteamOS `3.8.10` / `3.8.11` should now be resolved on the latest Stable release (`3.8.12` - https://store.steampowered.com/news/app/1675200/view/701020348316911441).
